### PR TITLE
DVC-7959 updated examples

### PR DESCRIPTION
**Pull request by chris-hoefgen**
- new local bucketing example
- revised cloud client example (rename, and cleanup)
- bug fix for cloud client track() method

### CODE REVIEW

File: devcycle_python_sdk/models/event.py [hunk 1]
```diff
@@ -22,15 +22,19 @@ class Event:
...
             for key in self.__dataclass_fields__
             if getattr(self, key) is not None and key != "date"
         }
         if self.date:
-            # convert to UTC and format as ISO string
-            json_dict["date"] = self.date.astimezone(tz=timezone.utc).isoformat()
+            if use_bucketing_api_format:
+                # convert to timestamp in milliseconds
+                json_dict["date"] = int(self.date.timestamp() * 1000)
+            else:
```
Comment (rob-odwyer) [inline devcycle_python_sdk/models/event.py:35]: Might be good to add an assertion in the bucketing client tests for the JSON we emit. I think I forgot that, which probably would have warned us about the difference here.

File: example/local_bucketing_client_example.py
Comment (rob-odwyer) [inline example/local_bucketing_client_example.py:1]: What do you think about calling both of these example clients from Github Actions? I'm thinking something really minimal, like a valid SDK key stored in a secret, associated with an account that doesn't necessarily even have these variables set up, and making sure the examples both have a zero exit code when run. Just to make sure they aren't embarrassingly broken going forward.

**Review by rob-odwyer — APPROVED** [review]

File: example/local_bucketing_client_example.py
Comment (chris-hoefgen) [reply example/local_bucketing_client_example.py:1]: that's not a bad idea. Any worry about that SDK key being fragile?  i wonder if there is a common one we can guarantee to exist?

Comment (chris-hoefgen) [reply example/local_bucketing_client_example.py:1]: Maybe we'll push that to a separate task

Comment (rob-odwyer) [reply example/local_bucketing_client_example.py:1]: I think if the SDK key was deleted, we'd have a test failure that was pretty easy to debug? Doesn't seem like a huge risk to me.

File: devcycle_python_sdk/models/event.py (hunk 1 above)
Comment (chris-hoefgen) [reply devcycle_python_sdk/models/event.py:35]: good call.

File: example/local_bucketing_client_example.py
Comment (chris-hoefgen) [reply example/local_bucketing_client_example.py:1]: created a new ticket for this one

**Pull request merged**